compiler-rt/lib/asan/asan_allocator.h

```diff
@@ -214,7 +214,7 @@ const uptr kAllocatorSpace = 0x600000000000ULL;
 const uptr kAllocatorSize  =  0x40000000000ULL;  // 4T.
 typedef DefaultSizeClassMap SizeClassMap;
 #  else
-const uptr kAllocatorSpace = 0x500000000000ULL;
+const uptr kAllocatorSpace = ~(uptr)0;
```

Comment:
looks like all but Apple, 

maybe include Apple, with apple reviewers, and move out constant from #ifdefs ?

Reply:
@jroelofs @wrotki Could you please take a look whether it would be ok to apply the dynamic allocator change to Apple as well? Thanks!

Reply:
ping @wrotki

Reply:
Late here - but it looks fine to me

Reply:
Thanks @wrotki!